packages/ifpstools.vm/ifpstools.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ifpstools.vm</id>
+    <version>2.0.2</version>
+    <authors>Wack0, zc, Jonson Tan</authors>
```

Comment:
@Wack0 we are adding IFPSTools.NET to flare-vm (`ifpsdasm` is very using useful to analyze InnoSetup!). I am not sure who are the tool's authors. Is `Wack0, zc, Jonson Tan`ok or should we use something different?

Reply:
Main author is myself, `ifpscc` is a fork of a .NET C compiler by Jonson Tan, hence the additional credit.

Reply:
btw, I didn't know about Inno Setup Decompiler before, looks like an interesting tool (although it doesn't support `push` opcode and expects every script to have an entrypoint function, so anything hand-optimised or compiled by `ifpscc` won't work with it).

Decompiler and debugger is on my to-do list for IFPSTools.NET eventually (will probably end up doing debugger first, and I would prefer to emit C for any hypothetical decompiler rather than pascalscript...)

Reply:
@Wack0 thanks for the answer! 😄  Should we then use `Wack0, Jonson Tan` to give you credit to both of you? Should we use `Wack0` for you? or `Rairii`? we can use whatever you prefer 😅